package.json

```diff
@@ -6,7 +6,7 @@
     "stylelint": "stylelint app/assets/stylesheets",
     "test": "jest",
     "build": "esbuild app/javascript/*.* --bundle --loader:.gif=dataurl --sourcemap --minify --outdir=app/assets/builds --public-path=assets --target=es6",
-    "build:css": "sass ./app/assets/stylesheets/application.scss:./app/assets/builds/application.css --style=compressed --load-path=node_modules --quiet-deps",
+    "build:css": "sass ./app/assets/stylesheets/application.scss:./app/assets/builds/application.css --style=compressed --load-path=node_modules --quiet-deps --silence-deprecation slash-div --silence-deprecation import",
```

Comment:
I agree with the cons, however as those warnings are not actionable by us and are cluttering our output, then I am ok with this, plus, it allows us to focus on warnings and errors in our own code, making it easier to debug and maintain.